designer/server/src/main/scala/pl/touk/nussknacker/ui/statistics/ScenarioStatistics.scala

```diff
@@ -26,16 +26,47 @@ object ScenarioStatistics {
 
   private val componentStatisticPrefix = "c_"
 
+  private val emptyScenarioStatistics: Map[String, String] = Map(
```

Comment:
I have created a couple of empty maps that would be used in case there are no scenarios. This way maps are created next to according methods, but on the other hand we could create one bigger map with all statistics we want to send every time

Reply:
It's ok if we have test which will verify if add new statistic entry and this new statistic is also added to emptyStatistics map.